docs/content/docs.md

```diff
@@ -1722,7 +1722,7 @@ Configuration Encryption
 ------------------------
 Your configuration file contains information for logging in to 
 your cloud services. This means that you should keep your 
-`.rclone.conf` file in a secure location.
+`rclone.conf` file in a secure location.
```

Comment:
rclone by default looks in `~/.config/rclone.conf` then in `~/.rclone.conf` (at least on Unix)

Reply:
Yes, but `.rclone.conf` is documented as the "older" variant so though its best to use the current name when referring to just one config file name..

> .config/rclone/rclone.conf (or .rclone.conf if created with an older version).

(https://rclone.org/docs/#config-config-file)